src/agentscope/service/web/search.py

```diff
@@ -194,3 +193,370 @@ def google_search(
             for result in results
         ],
     )
+
+
+def dblp_search(
```

Comment:
Please split this service function into three search functions for publication, author and venue.